tools/functional-tester/etcd-tester/key_stresser.go

```diff
@@ -140,13 +140,17 @@ func (s *keyStresser) run(ctx context.Context) {
 	}
 }
 
-func (s *keyStresser) Cancel() {
+func (s *keyStresser) Pause() {
 	s.cancel()
 	s.conn.Close()
 	s.wg.Wait()
 	plog.Infof("keyStresser %q is canceled", s.Endpoint)
 }
 
+func (s *keyStresser) Close() {
+	s.Pause()
```

Comment:
This is backwards since Pause is calling conn.Close. Put the Pause code in here and have Pause call Close instead